api/src/types/query.ts

```diff
@@ -36,4 +36,8 @@ export type FilterOperator =
 	| 'null'
 	| 'nnull'
 	| 'empty'
-	| 'nempty';
+	| 'nempty'
+	| 'intersects'
+	| 'nintersects'
+	| 'bbox_intersects'
+	| 'bbox_nintersects';
```

Comment:
There is a mismatch between these and the filter operators defined in `shared/types`. Is this type even used anywhere?

Reply:
I believed it's something I changed in a previous commit and missed when merging to the new release where these were moved to the `shared` package. This is most likely outdated and not used anywhere.